packages/storefront-lwc/src/modules/commerce/basketTotals/basketTotals.js

```diff
@@ -5,9 +5,10 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import { LightningElement, api } from 'lwc';
-import { ShoppingBasket } from 'commerce/data';
 
 export default class BasketTotals extends LightningElement {
+    eventChanged = false;
```

Comment:
Where is this `eventChanged` being used?

Reply:
Good catch, dont know where that came from. Removing it